community/best_practices_managing_issues.md

```diff
@@ -0,0 +1,58 @@
+# Best practices for managing onnx issues
+
+## Enforce templates
+Currently there are no templates for creating issues in onnx. This makes it very difficult for community members to triage issues. Going forward templates shall be added and maintained for ONNX issues.
+
+Following templates shall be added
+* ONNX Bug
+* ONNX Question
+* Propose New Feature, Operator or Enhancement
+
+
+## Label issues
+
+* Going forward all issues must be labeled. 
+* Multiple labels should be used to properly capture the issue. For example when a user logs a shape inference bug for a particular operator.  [bug][shape-inference] labels should be added.
+* Labels should use a consistent naming scheme
+* New label must be created for each release and it must be applied to all PRs and issues which are targeted for the release.
+* "backlog" label should be applied to all the issues which are cut from current release but are still on the radar to be addressed. This will prevent them from being closed as stale.
+
+### Rules for label naming scheme
+
+* Only lower case alphabets must be used
+* Whenever there are multiple words they must be separated by a hyphen '-'.
+* Names must be descriptive
+* Label for release must begin with "rel-"
+
+### Label tags
+
+Following labels will be added. [Some of these labels already exists]
```

Comment:
What is the criteria/definition for some of these? (i.e. needs-triage or tech-debt?)

Reply:
I had added these (needs-triage or tech-debt) as part of my initial draft, meant to remove it. Apart from these rest are pretty self explanatory.